app/view_models/auction_show_view_model.rb

```diff
@@ -73,7 +73,9 @@ def bid_status_class
   end
 
   def bid_status_partial
-    if auction.type == 'reverse' && (over? || available_and_user_is_winning_bidder?)
+    if reverse_auction_over? ||
```

Comment:
I don't know for sure. When I changed the names of the auction types, it was annoying because these factories started failing silently around the place and it bother. That said, it was a rare thing to do, but it felt kinda smelly